src/deluge/model/drum/kit.h

```diff
@@ -50,6 +50,8 @@ class Kit final : public Instrument, public GlobalEffectableForClip {
 	                  int32_t* reverbBuffer, int32_t reverbAmountAdjust, int32_t sideChainHitPending,
 	                  bool shouldLimitDelayFeedback, bool isClipActive);
 	void notifySamplesInterruptsSuspended();
+	inline void offerReceivedCCToModControllable(MIDIDevice* fromDevice, uint8_t channel, uint8_t ccNumber,
+	                                             uint8_t value, ModelStackWithTimelineCounter* modelStack);
```

Comment:
`inline` functions require the definition to be available at point of use, not just declaration. I think that probably happens in practice because this only gets used in `kit.cpp`, but using it like this in the header feels a bit odd.

Reply:
Yeah fair point, and that won't apply once it gets called by midi follow. I'll remove the inline since this is non critical code